test/cartesian/ReferenceLine/ReferenceLine.spec.tsx

```diff
@@ -354,18 +345,21 @@ describe('<ReferenceLine />', () => {
     expect(container.querySelectorAll('.recharts-reference-line-line')).toHaveLength(0);
   });
 
-  test('does not return anything when there is a duplicated category', () => {
+  it('should render one line when there is a duplicated category', () => {
     const firstDataItem = data[0];
     const dataWithDupe = [...data, firstDataItem];
     const { container } = render(
       <BarChart width={1100} height={250} data={dataWithDupe}>
         <XAxis dataKey="name" />
         <YAxis />
-        {/* reference line is currently unable to differentiate between duplicates on categorical axes. Which one do I render on? The first? Both? */}
         <ReferenceLine y={firstDataItem.name} ifOverflow="extendDomain" />
       </BarChart>,
     );
-    expect(container.querySelectorAll('.recharts-reference-line-line')).toHaveLength(0);
+    const allLines = container.querySelectorAll('.recharts-reference-line-line');
+    expect(allLines).toHaveLength(1);
+    // Is this correct? That looks like a category, not pixel coordinate.
+    expect(allLines[0]).toHaveAttribute('y', '201102');
```

Comment:
That's weird. Pixel values should be in the 100s. 

Maybe something to look into